app/assets/stylesheets/vendor/_select2.sass

```diff
@@ -0,0 +1,422 @@
+.select2-container
```

Comment:
This is default stylesheet for a JS lib, that's why it could go to
vendor/assets. Overrides (if any) should be placed in app/assets.

Application.sass is the "root" or so called manifest file for asset
pipeline. There are two ways to include other files - via @import (which
must be used for sass/scss) or via comment (e.g. *= dataTables.bootstrap).
The other way can be used for css but must not be used for sass/scss, so it
actually shouldn't be used for select2 right now (I missed that one in PR).



2015-04-02 8:08 GMT+02:00 bchucherko <notifications@github.com>:

> In app/assets/stylesheets/vendor/_select2.sass
> <https://github.com/eSpectrum-IT/brassica/pull/104#discussion_r27634334>:
>
> > @@ -0,0 +1,422 @@
> > +.select2-container
>
> OK.
> I don't know where to find
> of app where css files (bootstrap, application, select) are loaded.
> Furthermore I'm not sure how sass compilation works right now - it seems
> only application.sass is compiled - thus also this file (select3.sass)
> would have to be added to compilation as well or it can be compiled
> manually once (if it's more like a lib).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eSpectrum-IT/brassica/pull/104/files#r27634334>.
>



-- 
Tomasz Szymczyszyn